pkg/events/derive/so_export_symbol.go

```diff
@@ -0,0 +1,141 @@
+package derive
+
+import (
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/events/parse"
+	"github.com/aquasecurity/tracee/pkg/utils/sharedobjs"
+	"github.com/aquasecurity/tracee/types/trace"
+	"path"
+	"strings"
+)
+
+func SharedObjectExportWatchedSymbols(soLoader sharedobjs.DynamicSymbolsLoader, watchedSymbols []string, whitelistedLibsPrefixes []string) events.DeriveFunction {
+	gen := initSOExportWatchedSymbolsEventGenerator(soLoader, watchedSymbols, whitelistedLibsPrefixes)
+	return singleEventDeriveFunc(events.SymbolsLoad, gen.GenerateDeriveArgsFunc)
+}
+
+// Most specific paths should be at the top, to prevent bugs with iterations over the list
+var knownLibrariesDirs = []string{
```

Comment:
In my opinion this shouldn't be set externally.
It might be different between distributions, but we can just add these directories to this list.
It won't be a problem because different distributions might have new directories under the current one ("/usr/lib/x86_64-linux-gnu/" for example), not entirely new directories from what I know.
The only thing that might be of concern is security issues because of whitelist directory not existing in the system, but because it is a sub-directory of the normal lib directories it should be fine.
If you have example for a case that this isn't applied to please shared it and I will try to find other solution.